facility-manager.html

```diff
@@ -0,0 +1,806 @@
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+
+<p>More complex analysis can be done using the OMERO API that allows scripts written in Python, Matlab and other languages to use data from the OMERO server and save results back to the server.</p>
+
+<p>Further details on writing and installing scripts, the OMERO script repository and where to find other scripts are in <a href="http://www.openmicroscopy.org/site/community/scripts" target="_blank" class="external-url">Script Sharing</a> section  of the developer documentation.</p>
+
+<p>Further details of using MATLAB and Python in conjunction with OMERO are in the <a href="http://www.openmicroscopy.org/site/support/omero5.2/developers/scripts/matlab-scripts.html" target="_blank" class="external-url">Matlab and Python</a> section of the developer documentation.</p>
+
+<p>Details on <a href="https://www.openmicroscopy.org/site/support/omero5.2/developers/index.html#using-the-omero-api" target="_blank" class="external-url">Using the OMERO API</a> are in the OME developer documentation.</p>
+
+<p>OMERO.mtools is a suite of MATLAB-based tools which allow you to perform common image analysis tasks on images stored in an OMERO server. OMERO.mtools has its own GUI and only requires an installation of MCR (MATLAB Compiler Runtime) to run. Further details can be found on the <a href="http://www.openmicroscopy.org/site/products/partner/omero.mtools" target="_blank" class="external-url">OMERO.mtools</a> page of the OME website.</p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<h2><a class="anchor" id="imagej"></a>ImageJ and Fiji</h2>
+
+<p>Using the OMERO.insight plugin, ImageJ and Fiji can be used to open and view images on the OMERO server, draw ROIs and save the ROIs back to the OMERO server. Results of analysis performed in ImageJ or Fiji can be saved back to the OMERO server, attached as a file to the associated image. Full details of how to install the OMERO.insight plugin, and the workflows for using ImageJ and Fiji with OMERO are in the <a href="http://help.openmicroscopy.org/imagej.html" target="_blank" class="external-url">Using ImageJ with OMERO</a> section of the User Help.</p>
+
+<p class="top_link"><a href="#top"><img src="images/back_to_top.png" alt="Image for link to top of page" style="float: right;"/></a></p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<h2><a class="anchor" id="disk"></a>Disk usage</h2>
+
+<p>System administrators can view disk usage using the OMERO.web Admin tab. The disk usage reported reflects only data owned by the users or groups. Files that are uploaded to the server, or created in situ, and attached to Projects, Datasets or Images, are not included in the disk usage statistics.</p>
```

Comment:
Ah ok, thanks. Would certainly be better to be more explicit about what is and isn't included I think.